src/IO/Radio.cpp

```diff
@@ -95,49 +95,66 @@ void Radio::dequeueToRadio()
 
 void Radio::sendData(const sensorsData &currentState)
 {
-#if USE_LOGGER == 1
-	serialPrintf(fd, std::to_string(Logger::working).c_str());
-	serialPrintf(fd, ";");
-#endif
 	serialPrintf(fd, std::to_string(currentState.timeStamp).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
+	serialPrintf(fd, std::to_string(currentState.currentStateNo).c_str());
+	serialPrintf(fd, ",");
+
 #if USE_SBG == 1
 	serialPrintf(fd, std::to_string(currentState.sbg.roll).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.pitch).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.yaw).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.rollAccuracy).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.pitchAccuracy).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.yawAccuracy).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.gpsLatitude).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.gpsLongitude).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.gpsAltitude).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.relativeBarometricAltitude).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.velocityN).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.velocityE).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.velocityD).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.filteredXaccelerometer).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.filteredYaccelerometer).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.filteredZaccelerometer).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
 	serialPrintf(fd, std::to_string(currentState.sbg.solutionStatus).c_str());
-	serialPrintf(fd, ";");
+	serialPrintf(fd, ",");
```

Comment:
Could we replace this with an array + loop? It will not change anything functionality-wise, but maybe make this chunk of code more pretty?

Reply:
I guess we could have a fancy way to get all the data inside a structure. However, I was thinking of something simple like this:

```c++
std::string data[] = {
        std::to_string(currentState.sbg.roll),
        std::to_string(currentState.sbg.pitch),
        ...
};

for (auto i : data) {
       serialPrintf(fd, i.c_str());
       serialPrintf(fd, ";");
}
```